docs/pages/machine-id/access-guides/applications.mdx

```diff
@@ -1,204 +1,111 @@
 ---
 title: Machine ID with Application Access
-description: How to use Machine ID to access Applications
+description: How to use Machine ID to access applications
 ---
 
 (!docs/pages/includes/machine-id/v2-config-warning.mdx!)
 
-This guide demonstrates how to configure an automated service to use
-Machine ID to access a web application protected by Teleport.
+Teleport Application Access protects and controls access to HTTP and TCP
+applications. Machine ID can be used to grant machines secure, short-lived
+access to these applications.
 
-With Machine ID, Teleport issues short-lived certificates, tied to a machine
-identity, that can be rotated, audited, and managed with the same access
-controls that Teleport provides for human users.
+In this guide, you will configure `tbot` to produce credentials that can be
+used to access an application configured in Teleport Application Access.
 
 ## Prerequisites
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
 - If you have not already connected your application to Teleport, follow
   the [Application Access Getting Started Guide](../../application-access/getting-started.mdx).
-- If you're not already familiar with Machine ID, follow the
-  [Getting Started Guide](../getting-started.mdx) to familiarize yourself with
-  Machine ID. You'll also need `tctl` access to initially configure the bot.
 - (!docs/pages/includes/tctl.mdx!)
-- Ensure the `tbot` binary is installed on your Machine ID client system.
-  The client system is any system from which you want to access your Teleport
-  cluster and the resources it protects from an automated service. Refer to our
-  [Installation guide](../../installation.mdx) for instructions on installing
-  Teleport, which includes the necessary `tbot` binary.
+- `tbot` must already be installed and configured on the machine that will
+  access applications. For more information, see the
+  [platform guides](../platform-guides.mdx).
 
-## Step 1/3. Create a Machine ID bot and assign permissions
+## Step 1/3. Configure RBAC
 
-In this example you'll start by creating a bot named "example" and assign it a
-role granting access to some set of apps by label.
+First, Teleport must be configured to allow the credentials produced by the bot
```

Comment:
Same comment as [this one](https://github.com/gravitational/teleport/pull/32741/files#r1344546883) in the Ansible guide

Reply:
65205ceaea